python/paddle/distribution.py

```diff
@@ -672,13 +670,13 @@ class Categorical(Distribution):
 
             paddle.seed(100) # on CPU device
             x = paddle.rand([6])
-            print(x.numpy())
+            print(x)
             # [0.5535528  0.20714243 0.01162981
             #  0.51577556 0.36369765 0.2609165 ]
 
             paddle.seed(200) # on CPU device
             y = paddle.rand([6])
-            print(y.numpy())
+            print(y)
```

Comment:
The format of `print(y)` is not right. Same for other places.

Reply:
done

Reply:
dont't need to modify